### PR TITLE
[BEAM-14375] Fix Java Wordcount Dataflow postcommit for Gradle 7.4

### PR DESCRIPTION
**Pull request by Abacn**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Abacn** [comment]
Run Java Wordcount Dataflow

**Comment by Abacn** [comment]
Run Java_Examples_Dataflow PreCommit

**Comment by Abacn** [comment]
Tested workflow on my forked repo and passed: https://github.com/Abacn/beam/actions/runs/2240012578

Also included a workaround on java_actions.yml due to https://github.com/actions/runner/issues/1019 . Otherwise not able to initiate the workflow manually.

**Comment by Abacn** [comment]
R: @kennknowles
R: @robertwb

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -1802,6 +1800,7 @@ class BeamModulePlugin implements Plugin<Project> {
                 "--region=${dataflowRegion}"
               ])
             } else {
+              project.evaluationDependsOn(":runners:google-cloud-dataflow-java:worker:legacy-worker")
```
Comment (kennknowles) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:1803]: Ah, nice.

File: .github/workflows/java_tests.yml [hunk 2]
```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Run :sdks:java:core:test
         uses: ./.github/actions/gradle-command-action
         with:
-          arguments: :sdks:java:core:test
+          arguments: -p sdks/java/core/ test
```
Comment (kennknowles) [inline .github/workflows/java_tests.yml:77]: This is different? That makes me sad ;_;

**Review by kennknowles — APPROVED** [review]
Nice! Thanks for figuring it out!

**Pull request merged**